### PR TITLE
Adds a intra-round win/death log to an asteroid structure | Pod Wars

### PR DESCRIPTION
**Pull request by Zonespace27**
[QoL] [Feature]
## About the PR 
This adds an NT/SY win & death counter to a new structure, half-buried in an asteroid in the Pod Wars map. [Mostly Up-to-Date Images](https://imgur.com/a/k9MVirB)

## Why's this needed? 
Stats are fun, and having a scoreboard for the wins of each side will be nice to have, both for balance, and out of interest.

## Changelog

```changelog
(u)Zonespace
(*)Pod Wars team wins and deaths are now tracked, with a structure containing the stats being half-buried in an asteroid, near the Reliant.
```

### CODE REVIEW

**Comment by Gannets** [comment]
Kyle was just chatting a out adding this or something similar.

**Comment by Kyle2143** [comment]
Hah yeah. Sord just this morning mentioned that we should add something like this. Inspirations from the Cairgorn Memorial.

We were choosing between these 3 options:

1. Put a "Memorial" on each base ship telling them how many rounds your team won. (Without telling you about the rounds you lost) Could also do a "total enemy body count" on it too, as a reference to the US Army's policy of announcing enemy "Body Count" during the Vietnam War.
2. Put a memorial on each base ship telling em both teams win/loss ratio like normal.
3. Put a "Pod Wars" memorial somewhere on the map that has info for both sides, win count and death count.

I think we landed on number 3. Where sord would make a little satellite somewhere on the map to put a memorial. But I'm down to put maybe a board with just the one side on each ship. That has a Star Trek: DS9 Dominion war to me. Also, maybe real war too, I don't know that much about it.

**Comment by Kyle2143** [comment]
Also, that merge conflict is my fault. I desperately need to break up my megafile, it was almost acceptable when it was just me, but no.

Hopefully I'll have time to do that tomorrow night.

**Comment by Zonespace27** [comment]
@Kyle2143 Were you thinking about something similar to [this](https://imgur.com/a/k9MVirB)?

**Comment by Kyle2143** [comment]
> @Kyle2143 Were you thinking about something similar to [this](https://imgur.com/a/k9MVirB)?

Actually yeah, that's pretty much exactly what I was thinking of.

**Comment by BatElite** [comment]
You should probably specify in the changelog that you're talking about pod wars

**Comment by Zonespace27** [comment]
Sprites for the memorial unchanged but I'm putting it as ready regardless.

**Review by Kyle2143 — APPROVED** [review]

File: code/datums/gamemodes/pod_wars.dm [hunk 1]
```diff
@@ -381,8 +381,22 @@ ABSTRACT_TYPE(/datum/ore_cluster)
...
+	if (get_pod_wars_team_num(M) == TEAM_NANOTRASEN)
+		world.save_intra_round_value("nt_death", nt_death + 1)
+	if (get_pod_wars_team_num(M) == TEAM_NANOTRASEN)
+		world.save_intra_round_value("sy_death", sy_death + 1)
 	if (M.mind == our_team.commander)
 		our_team.change_points(-2)
+		if (get_pod_wars_team_num(M) == TEAM_NANOTRASEN)
+			world.save_intra_round_value("nt_death", nt_death + 1)
+		if (get_pod_wars_team_num(M) == TEAM_NANOTRASEN)
+			world.save_intra_round_value("sy_death", sy_death + 1)
```
Comment (ZeWaka) [inline code/datums/gamemodes/pod_wars.dm:399]: bad teams

File: code/datums/gamemodes/pod_wars.dm [hunk 2]
```diff
@@ -907,6 +932,100 @@ proc/setup_pw_crate_lists()
...
+		onclose(user, "pod_war_stats_[src]")
+		return
+
+/obj/memorial_left
+	name = "Memorial Inscription"
+	desc = "A memorial to those dead, but not forgotten."
+	icon = 'icons/obj/32x64.dmi'
+	icon_state = "memorial_left"
+
+/obj/memorial_right
```
Comment (ZeWaka) [inline code/datums/gamemodes/pod_wars.dm:1023]: please subtype these, probably as decor - abstract parent of each would be nice

File: code/datums/gamemodes/pod_wars.dm [hunk 3]
```diff
@@ -907,6 +932,100 @@ proc/setup_pw_crate_lists()
...
+		var/sy_deaths = world.load_intra_round_value("sy_death")
+		if(isnull(sy_wins))
+			sy_wins = 0
+		if(isnull(sy_deaths))
+			sy_deaths = 0
+
+		src.add_dialog(user)
+		user.Browse(src.desc, "title=Mission Log;window=pod_war_stats_[src];size=300x300")
+		onclose(user, "pod_war_stats_[src]")
+		return
```
Comment (ZeWaka) [inline code/datums/gamemodes/pod_wars.dm:1015]: useless ret

File: code/datums/gamemodes/pod_wars.dm [hunk 4]
```diff
@@ -907,6 +932,100 @@ proc/setup_pw_crate_lists()
...
+		var/nt_deaths = world.load_intra_round_value("nt_death")
+		if(isnull(nt_wins))
+			nt_wins = 0
+		if(isnull(nt_deaths))
+			nt_deaths = 0
+
+		src.add_dialog(user)
+		user.Browse(src.desc, "title=Mission Log;window=pod_war_stats_[src];size=300x300")
+		onclose(user, "pod_war_stats_[src]")
+		return
```
Comment (ZeWaka) [inline code/datums/gamemodes/pod_wars.dm:974]: useless ret

**Pull request merged**